mlrun/serving/states.py

```diff
@@ -346,6 +346,53 @@ def to(
         parent._last_added = step
         return step
 
+    def set_flow(
+        self,
+        steps: list[Union[str, StepToDict, dict[str, Any]]],
+        force: bool = False,
+    ):
+        """set list of steps as downstream from this step, in the order specified. This will overwrite any existing
+        downstream steps.
+
+        :param steps: list of steps to follow this one
+        :param force: whether to overwrite existing downstream steps. If False, this method will fail if any downstream
+        steps have already been defined. Defaults to False.
+
+        example:
+            a 3-step pipeline (step1 -> step2 -> step3) to overwrite existing downstream steps:
+            step1.set_flow(
+                [
+                    dict(name="step2", handler="step2_handler"),
+                    dict(name="step3", class_name="Step3Class"),
+                ],
+                force=True,
+            )
+        """
+        if hasattr(self, "steps"):
+            parent = self
+        elif self._parent:
+            parent = self._parent
```

Comment:
I know this code is used also in the `to()` method, but what does it mean in this context? The `steps` attribute will exist in Flow steps, so what's happening in this part? Should this really apply to this scenario which should only work for Flow steps?

Reply:
According to the requirement
> this should be enabled for any step in the graph